fv3core/stencils/del2cubed.py

```diff
@@ -98,10 +99,10 @@ def compute(qdel: FloatField, nmax: int, cd: float, km: int):
 
     # Construct some necessary temporary storage objects
     fx = utils.make_storage_from_shape(
-        qdel.shape, origin=origin, cache_key="del2cubed_fx"
+        qdel.shape, origin=origin, cache_key="del2cubed_fx", init=True
     )
     fy = utils.make_storage_from_shape(
-        qdel.shape, origin=origin, cache_key="del2cubed_fy"
+        qdel.shape, origin=origin, cache_key="del2cubed_fy", init=True
```

Comment:
init=True comes with a performance cost, but is needed here? given how fast removing most of the syncs is, I am ok with this

Reply:
I will confirm that this is still required.

Reply:
This was no longer required :)